src/Components/BookDetails/BookDetails.tsx

```diff
@@ -90,7 +90,7 @@ const BookDetails = () => {
   }
 
   const borrowerOptions = () => {
-    return bookDetails?.users?.map((value: Value, index: number, array: Value[]): JSX.Element => {
+    return bookDetails?.users?.map((user: any): JSX.Element => {
```

Comment:
Thank you for this, that should allow all associated tests to pass.

Reply:
Oh perfect!